Sources/web3swift/Operations/ReadOperation.swift

```diff
@@ -16,7 +16,7 @@ public class ReadOperation {
     public var method: String
     public var data: Data? { transaction.data }
 
-    var resolver: PolicyResolver
+    var policyResolver: PolicyResolver
```

Comment:
Just a quick suggestion - please search across the project where the `PolicyResolver` is used to rename the variables from `resolver` to `policyResolver`. I've seen that some tests still use `resolver` as a name.
Let's make it as good as we can 😊

Reply:
This was actually the only one, where it used within the whole lib)